app/mailers/rent_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class RentMailer < ApplicationMailer
+
+  def rent_email(rent)
+    @user = rent.user
+    @rent = rent
+    @book = Book.find(rent.book_id)
```

Comment:
As you are doing with user, you could do the same with the book `@book = rent.book`